### ISSUE TITLE
Open Audit Log on an soft-deleted record results in an Exception

### ISSUE THREAD

**Issue by ludgerey** [dev-partners/laravel-nova-auditable-log#10]
Reproduce:
* Create a Model with SoftDeletes enabled
* Create a record in Nova
* Delete the record in Nova
* Set Filter to "With Trashed"
* Open record in Nova again
* Click on "Audit Log"

-> Nova will show an Exception:

```
Too few arguments to function Devpartners\AuditableLog\ToolServiceProvider::Devpartners\AuditableLog\{closure}(), 1 passed in [...]/Illuminate/Auth/Access/Gate.php on line 469 and exactly 2 expected
```

Sorry, I currently have no time to fix the issue by myself and create a PR.

**Comment by recycledbeans** [comment]
Hey @ludgerey thanks for reporting this. I'll see if I can create a PR today to resolve.

**Comment by recycledbeans** [comment]
Hey @ludgerey - I pushed a fix for this just now, which is available in v1.0.8 (make sure composer pulls in that latest version upon update).

**Comment by NickBelhomme** [comment]
still not fixed in 1.0.9

![image](https://user-images.githubusercontent.com/312003/114297121-27a4d400-9aaf-11eb-997f-086f9418b4ad.png)
